src/components/core/index.ts

```diff
@@ -1,7 +1,8 @@
 import { Component } from '../types.js';
 import { interactionHandler } from './interaction-handler.js';
+import { messageListener } from './message-listener.js';
 import { ready } from './ready.js';
 
 export default {
-    gatewayEvents: [ready, interactionHandler],
+    gatewayEvents: [ready, interactionHandler, messageListener],
```

Comment:
Let's move our message listener to the `qbox` component rather than `core`, as it's a Qbox-specific feature. Technically our bot can be added to other servers and I don't think we should include this functionality absolutely everywhere.

We should also name this something better such as `imageErrorRecognition`.

Reply:
Done.